File: AvernusGames_Front-End/src/app/models/ItemVenda.ts
import { Game } from "./Game";
import { Venda } from "./Venda";

export class ItemVenda {
    vendaId: number = 0;
    venda: Venda = new Venda();
    gameId: number = 0;
    game?: Game;
    qtd: number = 0;
  }
  

File: AvernusGames_Front-End/src/models/ItemVenda.cs
namespace Avernus_Games_Store.src.models
{
    public class ItemVenda
    {
        public int Id { get; set; }
        public Produto? Produto { get; set; }
        public int Qtd { get; set; }

        public ItemVenda() { }

        public ItemVenda(Produto produto, int qtd)
        {
            Produto = produto;
            Qtd = qtd;

        }


    }
}